src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfig.java

```diff
@@ -64,6 +68,16 @@ public VoxelSniperConfig(
         this.brushProperties = brushProperties;
     }
 
+    /**
+     * Return if update notifications are enabled.
+     *
+     * @return {@code true} if notifications for updates are enabled, {@code false} otherwise.
+     * @see VoxelSniperConfigLoader#areUpdateNotificationsEnabled()
```

Comment:
What is the reference for?

Reply:
All the other configuration options have a reference to VoxelSniperConfigLoader (trying to stay consistent)